packages/opensrp-plans/src/constants.tsx

```diff
@@ -58,5 +60,13 @@ export const TableColumnsNamespace = 'plans';
 
 /** route params for product-catalogue pages */
 export interface RouteParams {
-  plansId?: string;
+  planId?: string;
 }
+
+/** Enum representing the possible plan types colors*/
+export const PlanStatusColors: Dictionary = {
+  active: '#22B509',
+  complete: '#000000',
+  draft: '#DBA400',
+  retired: '#000000',
```

Comment:
@kahummer @p-netm would it be appropriate to have the colors come from a contants-like module named "colours"?  So that it would be possible to have GREEN here instead of '#22B509', for example

Reply:
@moshthepitt I agree